ipie/propagation/tests/test_generic.py

```diff
@@ -334,6 +329,35 @@ def test_vhs():
         assert numpy.allclose(vhs_batch[iw], vhs_serial[iw])
 
 
+@pytest.mark.unit
+def test_fp():
+    numpy.random.seed(7)
+    nmo = 10
+    nelec = (6, 5)
+    nwalkers = 8
+    nsteps = 25
+    qmc = dotdict(
+        {
+            "dt": 0.005,
+            "nstblz": 5,
+            "nwalkers": nwalkers,
+            "batched": False,
+            "hybrid": True,
+            "num_steps": nsteps,
+        }
+    )
+    qmc.batched = True
+    batched_data = build_test_case_handlers(nelec, nmo, num_dets=1, options=qmc, seed=7)
+    prop_fp = FreePropagation(time_step=0.005, verbose=False, ene_0=-1.0)
+    prop_fp.build(batched_data.hamiltonian, batched_data.trial)
+
+    prop_fp.propagate_walkers(
+        batched_data.walkers, batched_data.hamiltonian, batched_data.trial, -1.0
+    )
+    assert batched_data.walkers.phia.shape == (nwalkers, nmo, nelec[0])
```

Comment:
It's helpful in these tests to capture some output (like the walker overlap or something) for a fixed seed and test against those numbers so we catch regressions.

Reply:
added some asserts